sqlserver/assets/monitors/sqlserver_db_not_sync.json

```diff
@@ -6,7 +6,7 @@
   "tags": [
     "integration:sql-server"
   ],
-  "description": "Backups are essential for data protection, disaster recovery, and maintenance operations. This monitor tracks how synchronized your database is with its backup. Alerts help to avoid scenarios where backups are are incomplete, leaving your data at risk.",
+  "description": "Backups are essential for data protection, disaster recovery, and maintenance operations. This monitor checks if you database is marked for replication sync. Alerts help to avoid scenarios where databases are not synced, leaving your data at risk.",
```

Comment:
```suggestion
  "description": "Backups are essential for data protection, disaster recovery, and maintenance operations. This monitor checks if your database is marked for replication sync. Alerts help to avoid scenarios where databases are not synced, leaving your data at risk.",
```